src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[500];
```

Comment:
1. solution is correct now but for the moment task requires 3 balls
2. magic numbers should be extracted to constants with proper naming

Reply:
How to extract magic numbers into constants with their proper name?